### PR TITLE
feat(dashboards): Widget Library Issue Widget title and description update.

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Updates the Widget Library Issue Widget title and description reflect recent changes made to widget
image

### CODE REVIEW

**Review by shruthilayaj — COMMENTED** [review]
Some of the example use cases reference the `Latest Unresolved Issues` widget, can we update those too? Thanks Ed!

**Comment by edwardgou-sentry** [comment]
> Some of the example use cases reference the `Latest Unresolved Issues` widget, can we update those too? Thanks Ed!

Yup! Updated, thanks!

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ The library includes the following widgets:
 - **LCP by Country**: World map showing the p75 of page load times for each country
 - **Miserable Users**: The total number of unique users who have experienced slow transactions (transaction duration greater than 1200ms)
 - **Slow vs Fast Transactions**: Bar chart comparing the percentage of transactions that are over 300ms (slow) and under 300ms (fast)
-- **Latest Unresolved Issues**: A table of unresolved issues ordered by the most recently seen issues
+- **Issues For Review**: A table of unresolved issues for you or your team's review ordered by the most recently seen issues
```
Comment (shruthilayaj) [inline src/docs/product/dashboards/widget-library/index.mdx:22]: might have to be either `for your or your team's review` or `for you or your team to review`?

Comment (edwardgou-sentry) [reply src/docs/product/dashboards/widget-library/index.mdx]: english hard 😃 

updated to `for you or your team to review`!

**Pull request merged**